Digipost.Signature.Api.Client.Direct/Digipost.Signature.Api.Client.Direct.csproj

```diff
@@ -1,28 +1,28 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-        <Title>Digipost Signature Api Client Direct</Title>
-        <PackageId>Digipost.Signature.Api.Client.Direct</PackageId>
-    </PropertyGroup>
-
-    <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="2.*"/>
-        <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.*"/>
-        <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.*"/>
-        <PackageReference Include="System.Net.Requests" Version="4.*"/>
-        <PackageReference Include="System.Security.Cryptography.Xml" Version="4.*"/>
-        <PackageReference Include="api-client-shared" Version="4.0.0"/>
-
-        <PackageReference Include="Digipost.Signature.Api.Client.Core" Version="5.*"/>
-    </ItemGroup>
-
-    <ItemGroup>
-        <ProjectReference Include="..\Digipost.Signature.Api.Client.Core\Digipost.Signature.Api.Client.Core.csproj"/>
-    </ItemGroup>
-
-    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-        <DocumentationFile>bin\Release\netcoreapp2.1\Digipost.Signature.Api.Client.Direct.xml</DocumentationFile>
-        <noWarn>1591</noWarn>
-    </PropertyGroup>
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Ingen endringer her annet enn spaces. Virker som spacing har endret seg for configfiler i nye Rider, så lot den bare gå bananas.